### PR TITLE
WP-23 More unit tests

### PR DESCRIPTION
**Pull request by mmcgahan**
**`Blocked` until #57 merges**

**`Prelim` while I flesh out the tests - just want to get Travis running on it.**

### CODE REVIEW

File: src/util/serverUtils.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import https from 'https';
```
Comment (mmcgahan) [inline src/util/serverUtils.js]: moved these functions from `src/server.js` to this utils module to make it easier to mock them when testing the `server.js` code

Comment (sadafie) [inline src/util/serverUtils.js]: this is such a nitpick that's not related to this PR, but i think we talked about this once before, so I wanted to bring this up. Should we rename this directory from `util` to `utils` for consistency?

**Comment by sadafie** [comment]
@mmcgahan these changed look good, though the branch has conflicts. is this still `Prelim`?

**Comment by mmcgahan** [comment]
@sadafie yeah there are quite a few more tests I want to add, but a fair amount of the missing coverage is related to other tickets in my current sprint, so I wanted to build from that work in this branch - probably will stay Prelim until near the end of the sprint.

File: src/util/serverUtils.js (hunk 1 above)
Comment (mmcgahan) [inline src/util/serverUtils.js]: yes - that change has been made in the starter repo but hasn't been carried over yet

**Comment by mmcgahan** [comment]
There's a lot more testing I want to add, but I'm going to merge this for now so that it un-blocks some other stuff

More to come

**Pull request merged**